src/js/popup/popup.js

```diff
@@ -343,10 +343,12 @@
             }, false);
           
           // Get masks and determine what to display
-          const masks = await popup.utilities.getMasks(getMasksOptions);
+          const { relayAddresses } = await browser.storage.local.get(
+            "relayAddresses"
+          );
 
           // If no masks are created, show onboarding prompt
-          if (masks.length === 0) {
+          if (relayAddresses.length === 0) {
```

Comment:
TODO: There may be an edge case where a user initially has no masked created when using the add-on, and then creates a mask elsewhere (other browser, etc) and they get the empty state when they should see their current masks. 

This may be the one spot to live/refresh based on new data.

Reply:
Revised in dffd2674682978a1b97e460c2b67caf308196259

Moved this entire logic block to `buildMasksList()`

I ran some tests locally if a user creates a mask else where and this loads. It refreshes without issue. I also tested the inverse (local storage has one mask, but user deletes it elsewhere). All good!